frameworks/template/build.self.hosted.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
How about writing this file out from within `new-standalone-service.sh`? It's closely tied to the logic in there, could make sense to just emit it directly. See examples in `tools/vagrant/build-dcos-docker.sh`